devito/propagator.py

```diff
@@ -34,6 +34,7 @@ class Propagator(object):
     def __init__(self, name, nt, shape, spc_border=0, time_order=0,
                  time_dim=None, space_dims=None, forward=True, compiler=None,
                  profile=False, cache_blocking=False, block_size=5):
+        self.factorized = []  # to hold factored terms. gets set in operator
```

Comment:
Technically, this should be part of the constructor signature, so that people building raw Propagator instances can easily access this feature. But then again, the same could be said for the stencils, so we don't really need to fix this for this PR.